tests/regression/test_interpolate.py

```diff
@@ -204,6 +204,21 @@ def test_cell_orientation():
     assert abs(g.dat.data - h.dat.data).max() < 1e-2
 
 
+def test_cell_orientation_curve():
+    m = CircleManifoldMesh(3)
+    x = SpatialCoordinate(m)
+    m.init_cell_orientations(x)
+
+    U = VectorFunctionSpace(m, 'CG', degree=1)
+    V = VectorFunctionSpace(m, 'CG', degree=2)
+    f = project(CellNormal(m), U)
+
+    g = interpolate(f, V)
+    h = project(f, V)
+
+    assert abs(g.dat.data - h.dat.data).max() < 1e-2
```

Comment:
If we are testing if interpolation is working on 1D curve, I think the following is more concise:
```
m = CircleManifoldMesh(3)
x = SpatialCoordinate(m)
m.init_cell_orientations(x)
V = VectorFunctionSpace(m, 'DG', 0)
f = interpolate(CellNormal(m), V)
assert np.allclose(f.dat.data, [[1 / 2, sqrt(3) / 2],
                                [-1, 0],
                                [1 / 2, -sqrt(3) / 2]])
```

Reply:
Thanks for the suggestion. Done